README.md

```diff
@@ -2,7 +2,9 @@
 
 jmx2graphite is a one liner tool for polling JXM and writes into Graphite (every 30 seconds by default). You install & run it on every machine you want to poll its JMX.
 
-Currently it only reads JMX from a jolokia agent running on a JVM, since exposing JMX is the simplest and easiest through Jolokia agent (1 liner - see below).
+Currently it has two flavors.
+1. Reads JMX from a jolokia agent running on a JVM, since exposing JMX is the simplest and easiest through Jolokia agent (1 liner - see below).
+2. Run as a java agent, and get metrics directly from MBean Platform
```

Comment:
This is useful if your app is running inside Docker on Mesos and it coupling it with another container just to read its metrics contradicts the Mesos paradigm.